generators/go/sdk/CHANGELOG.md

```diff
@@ -7,11 +7,44 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- ## Unreleased -->
 
+## Unreleased
+
+- Fix: Package documentation is now generated into the correct package's `doc.go`.
+- Feature: Add support for generated endpoint snippets.
```

Comment:
nit: I think it will be useful to specify where the code snippets will show up and in this case it will be for the documentation and over the snippets API but not embedded in the SDK itself

Reply:
Great call, will update 👍